api/src/main/java/org/openmrs/Order.java

```diff
@@ -362,6 +363,41 @@ public void setCommentToFulfiller(String commentToFulfiller) {
 	}
 	
 	/**
+	 * Convenience method to determine if the order is activated as of the current date
+	 * 
+	 * @return boolean indicating whether the order was activated on the current date
+	 * @since 2.0
+	 * @should return true if an order was activated on the current date
+	 * @should return true if an order was activated before the current date
+	 * @should return false for an order not activated
+	 * @should return false for an order activated after the current date
```

Comment:
Then I think the wording needs to be clearer, one should say when date activated is null and the other should say when date activated is after the check date